py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -2133,6 +2133,26 @@ def aten_ops_linear(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten._cdist_forward.default)
+def aten_ops_cdist_forward(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.normalization.cdist_forward(
+        ctx,
+        target,
+        SourceIR.ATEN,
+        name,
+        x1=args[0],
+        x2=args[1],
+        p=args[2],
+        compute_mode=args[3],
```

Comment:
Based on the [schema](https://github.com/pytorch/pytorch/blob/7b11fb4695c61e04c7032e039f0a9f02636e0176/aten/src/ATen/native/native_functions.yaml#L4415) having `int?` as the type for `compute_mode`, this should be `compute_mode = args_bounds_check(args, 2, None)`